docs/iot/iot-partners/sdt_som.md

```diff
@@ -0,0 +1,71 @@
+# Tizen-enabled SOM [SDT-A7X2-M4]
+
+![Architecture](media/sdt.png)
+
+Gateway SOM based on Arm&reg; Cortex&reg;-A7 dual core and another Cortex&reg;-M4 core, where the former one is for high-performance processing and the latter one is for real-time application performance. It also provides 512MB DDR3L-1600 DRAM and 8GB eMMC NAND Flash. 
```

Comment:
"Copyright © 2019 Sigma Delta Technologies Inc. All rights reserved.
No part of this publication may be reproduced, stored in a retrieval system, or transmitted in any form or by any means – electronic, mechanical, photocopying, recording or otherwise – without the permission of Sigma Delta Technologies.
Trademarks and trade names may be used in this document to refer to either the entities claiming the marks and names or their products. Sigma Delta Technologies disclaims any proprietary interest in the trademarks and trade names other than its own."

As per Copyright &copy; 2019 Sigma Delta Technologies Inc., taking and rephrasing content from [Product Brief_SDT-A7X2-M4](https://docs.wixstatic.com/ugd/166c05_9b9635ed83d64bfeb7731fe0671fb511.pdf) may lead to plagiarism and legal issues (Even though we have provided the reference link).

@mijin85cho, @xbldx 
Please help clarify.

Reply:
I agree with @Reni-90's concern.
The starting point of this update is simply copying & paste of https://www.sigma-delta.tech/sdt-a7x2-m4. 
As @Reni-90 pointed out, it has the possibility of problem.

So, how about just removing from line 3 to line 49, and only providing Tizen only information, with reference link to SDT-A7X2-M4?
It would be better to remove the content from other site.

Reply:
Thanks for pointing out good point, @Reni-90.

Reply:
@mijin85cho 
Thank you very much.

As Sigma Delta Technologies Inc. is very strict with their content. I think we can give an introductory line and can remove the lines (3-49) as you suggested. We can also give a reference.

# Tizen-enabled SOM [SDT-A7X2-M4]

Tizen-enabled System-on-Module (SOM) [SDT-A7X2-M4] is a gateway SOM based on Arm® Cortex®-A7 dual core and Arm® Cortex®-M4 core.
For more information, see [SDT-A7X2-M4 and its specifications](https://www.sigma-delta.tech/sdt-a7x2-m4).

Reply:
As I discussed with Mijin Cho on messanger,
The initial version of this document is created by Sigma Delta.
Because they want to upload their contents to our sites, I think there is no issue about it.

Especially, this board is created for Tizen together. 
Sigma Delta + G.Camp(Seoul City) + Tizen Team